bluemix/configuration/core_config/bx_config.go

```diff
@@ -752,9 +754,44 @@ func (c *bxConfig) ClearSession() {
 		c.data.IsLoggedInAsCRI = false
 		c.data.ResourceGroup = models.ResourceGroup{}
 		c.data.LoginAt = time.Time{}
+		c.data.PaginationURLs = []models.PaginationURL{}
 	})
 }
 
+func (c *bxConfig) SetPaginationURLs(paginationURLs []models.PaginationURL) {
+	c.write(func() {
+		c.data.PaginationURLs = paginationURLs
+	})
+}
+
+func (c *bxConfig) ClearPaginationURLs() {
+	c.write(func() {
+		c.data.PaginationURLs = []models.PaginationURL{}
+	})
+}
+
+func (c *bxConfig) AddPaginationURL(index int, url string) error {
```

Comment:
Never returns `error` only `nil`. Looks like this one got missed on the last change.

Reply:
Thanks for catching that again. I am not sure why it wasn't caught by my editor. Pushed up changes